.eslintrc

```diff
@@ -2,5 +2,11 @@
   "extends": [ "apostrophe", "plugin:vue/vue3-recommended" ],
   "globals": {
     "apos": true
+  },
```

Comment:
Why transition back to the old format? https://eslint.org/docs/latest/use/configure/configuration-files

This file must be .eslintrc.json or .yml or something

Reply:
config has been renamed.